### PR TITLE
edge: add support for canTrickleIceCandidates

### PR DESCRIPTION
**Pull request by fippo**
as described in https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/canTrickleIceCandidates

### CODE REVIEW

**Comment by fippo** [comment]
wait a second, i'm rewriting the test...

File: test/unit/edge.js [hunk 1]
```diff
@@ -941,7 +966,6 @@ describe('Edge shim', () => {
           return pc.createAnswer();
         })
         .then((answer) => {
-          console.log(answer.sdp);
```
Comment (fippo) [inline test/unit/edge.js:944]: drive-by fix

**Comment by KaptenJansson** [comment]
Are the unhandled promises in the unit test related to this? 

https://travis-ci.org/webrtc/adapter/jobs/218795863#L1000-L1002

**Comment by fippo** [comment]
whoops. Should be an async test...

**Comment by KaptenJansson** [comment]
Right, you were missing done in the it statements. Fast fix :)

**Review by KaptenJansson — APPROVED** [review]
LGTM
